libcudacxx/include/cuda/std/__functional/not_fn.h

```diff
@@ -72,4 +73,12 @@ _LIBCUDACXX_HIDE_FROM_ABI _CCCL_CONSTEXPR_CXX20 auto not_fn(_Fn&& __f)
 
 _LIBCUDACXX_END_NAMESPACE_STD
 
+#if _CCCL_STD_VER > 2014
```

Comment:
Nitpick: I try very hard to move all our dialect conditionals towards a notation that includes the valid dialact
```suggestion
#if _CCCL_STD_VER >= 2017
```

Its easy now, but with later standards it will become more and more iffy